airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -51,6 +51,8 @@ class Meta:
 
     dag_id = auto_field(dump_only=True)
     dag_display_name = fields.String(attribute="dag_display_name", dump_only=True)
+    bundle_name = auto_field(dump_only=True)
```

Comment:
If the fields are added to legacy API, they should also be added to "new" fast API

Reply:
That's being done in #45644, plus, this will all be changing in #45665 anyways.